docs/build/configure/webhooks.md

```diff
@@ -11,35 +11,91 @@ Webhooks allow you to trigger actions when certain types of data are sent from y
 For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
 Viam provides two webhook types depending on the event you want to trigger on:
 
-- `"part_data_ingested"`: trigger when a sensor on the machine part collects a new reading
-- `"part_online"`: trigger when the machine part is online
+- Data has been synced to the cloud: trigger when data from the machine is synced
+- Part is online: trigger continuously at specified interval while the {{< glossary_tooltip term_id="part" text="machine part" >}} is online
+- Part is offline: trigger continuously at specified interval while the machine part is offline
```

Comment:
```suggestion
- **Part is offline**: trigger continuously at specified interval while the machine part is offline
```
+ above, suggest as UI element

Reply:
I was meaning it as conceptual instead of as a UI element, since we're not actually talking about the UI just yet